### PR TITLE
Abstract

### PR DESCRIPTION
**Pull request by pllim**
I took a stab at the abstract, mainly copying some wordings from the last paper. Does that mean I need to cite the last paper in the abstract? 😆

### CODE REVIEW

File: src/ms.tex [hunk 1]
```diff
@@ -53,9 +53,18 @@
...
-
+The \astropy Project supports and fosters the development of open-source and openly-developed
+\python packages that provide commonly-needed functionality to the astronomical
+community.
+A key element of the \astropy Project is the core package \astropypkg, which serves as the
+foundation for more specialized projects and packages.
+In this article, we summarize key features in the core package as of the recent major
+release, version 5.0, and provide major updates for the project.
+We then discuss supporting a broader ecosystem of inter-operable packages,
+including connections with several observatories and missions.
```
Comment (adrn) [inline src/ms.tex:64]:
```suggestion
including connections with several astronomical observatories and missions.
```

File: src/ms.tex [hunk 2]
```diff
@@ -53,9 +53,18 @@
...
+We also revisit the future outlook of the \astropy Project and the current
+status of Learn Astropy. Last but not least, we contemplate the current
+and future challenges facing the project.
```
Comment (adrn) [inline src/ms.tex:67]:
```suggestion
status of Learn Astropy. 
We conclude by raising and discussing the current and future challenges facing the project.
```

**Review by adrn — COMMENTED** [review]
Looks great! Only considering some minor tweaks

**Comment by pllim** [comment]
Applied your suggestions, @adrn . Thanks for the review!

What is the merge policy here?

**Comment by adrn** [comment]
> What is the merge policy here?

YOLO

**Pull request merged**